File: ReponseAppelOffres/Server/Database/Models/Message.cs
using System;

namespace Server.Database.Models
{
    public class Message
    {
        public string SenderId { get; set; }
        public string SenderUsername { get; set; }
        public string Channel { get; set; }
        public string Content { get; set; }
        public DateTime Timestamp { get; set; }
    }
}

File: ClientLegerV2/app/src/main/java/polymtl/equipe3/clientlegerv2/model/Message.kt
package polymtl.equipe3.clientlegerv2.model

import com.google.gson.annotations.SerializedName
import java.util.*


data class Message(

    @field:SerializedName("senderId")
    val SenderId: String?,
    @field:SerializedName("senderUsername")
    val SenderUsername: String?,
    @field:SerializedName("channel")
    val Channel: String?,
    @field:SerializedName("content")
    val Content: String?,
    @field:SerializedName("timestamp")
    val Timestamp: Date
)

